aspnetcore/mvc/views/overview.md

```diff
@@ -64,14 +64,14 @@ When an action returns a view, a process called *view discovery* takes place. Th
 
 When an action returns the `View` method, like so `return View();`, the action name is used as the view name. For example, if this were called from an action method named "Index", it would be equivalent to passing in a view name of "Index". A view name can be explicitly passed to the method (`return View("SomeView");`). In both of these cases, view discovery searches for a matching view file in:
 
-   1. Views/<ControllerName>/<ViewName>.cshtml
+   1. Views/\<ControllerName>/\<ViewName>.cshtml
 
-   2. Views/Shared/<ViewName>.cshtml
+   2. Views/Shared/\<ViewName>.cshtml
 
 >[!TIP]
 > We recommend following the convention of simply returning `View()` from actions when possible, as it results in more flexible, easier to refactor code.
 
-A view file path can be provided, instead of a view name. In this case, the *.cshtml* extension must be specified as part of the file path. The path should be relative to the application root (and can optionally start with "/" or "~/"). For example: `return View("Views/Home/About.cshtml");`
+A view file path can be provided, instead of a view name. If using an absolute path starting at the application root (optionally starting with "/" or "~/"), the *.cshtml* extension must be specified as part of the file path. For example: `return View("Views/Home/About.cshtml");`. Alternatively, one can use a relative path from the controller-specific directory within the *Views* directory to specify views in different directories (e.g. within *HomeController.cs*: `return View("../Manage/Index");`) or, redundantly, views within the current controller-specific directory (e.g. within *HomeController.cs*: `return View("./About");`). Note that relative paths do not use the *.cshtml* extension. In general, as previously mentioned, it is best practice to have the file structure for views reflect the relationships between controllers, actions, and views for purposes of maintainability and clarity.
```

Comment:
Oh ... I missed the paragraph! lol ... Irma is throwing me off a bit (I'm in Florida).

* Change your "e.g." to "for example" ... It's a guideline: I think because they don't translate well.
* Instead of third person "one," go with second person "you." 
* Use contractions, so change the "do not" to "don't." Later, change "it is" to "it's."
* "In general, as previously mentioned ..." is a comma splice, but the guidelines ask authors not to start sentences with two prepositional phrases. Therefore, it's prob best to just start it with "As previously mentioned, ..."
* Last sentence: Use "among" ... "between" is only for connecting pairs of things.
* The last sentence is correct, but it can be rewritten from a passive "it's best to" into an action with an unstated "you should." That's somewhat optional. The guidelines prefer active voice, but they don't absolutely demand it of every bleed'in sentence. lol [@]Rick-Anderson also has me well-trained now to be on the lookout for longer-ish sentences and to try and chop them down.

> As previously mentioned, follow the best practice of organizing the file structure for views to reflect the relationships among controllers, actions, and views for maintainability and clarity.